src/Reflex/Todo.hs

```diff
@@ -1,19 +1,30 @@
 {-# LANGUAGE RecursiveDo, ScopedTypeVariables, FlexibleContexts, TypeFamilies, ConstraintKinds, TemplateHaskell #-}
+module Reflex.Todo where
 
 import Prelude hiding (mapM, mapM_, all, sequence)
 
+import Data.Map (Map)
+import qualified Data.Map as Map
 import Reflex
 import Reflex.Dom
 
+insertNew_ :: (Enum k, Ord k) => v -> Map k v -> Map k v
+insertNew_ v m = case Map.maxViewWithKey m of
```

Comment:
Good question, I don't see why it should have the underscore either. I was following along the repo linked in the project, and they called it that... but I don't see any reason it should be.